### PR TITLE
Adding blog post talking about new CVEs and security team response

### PR DESCRIPTION
**Pull request by breadchris**
A blog post was warranted to discuss the new CVEs and how security teams should consider them. It is too tempting to just continually update to the latest version, but understanding the cost is also important.

### CODE REVIEW

File: docs/blog/2021-12-28-log4j-hype-train.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+A public [pull request](https://github.com/apache/logging-log4j2/pull/608) was created to fix the Log4Shell vulnerability.
+It is important to note that a CVE has [not been created yet](https://nvd.nist.gov/vuln/detail/CVE-2021-44228) and the vulnerability
+is not very widely known about outside this PR.
+
+### [12/10/21] CVE Assigned for Log4Shell
+
+The National Vulnerability Database [assigns the Log4Shell vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2021-44228) the identifier CVE-2021-44228 and is given a severity of 10.0.
+
+### [12/11/21] 2.15.0 Released
```
Comment (freeqaz) [inline docs/blog/2021-12-28-log4j-hype-train.mdx:105]: I thought this was released on the 6th?

File: docs/blog/2021-12-28-log4j-hype-train.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+vulnerabilities and leaves it up to developers to decipher an alert like: "CVE-2021-44832 was found in your code, fix it now!".
+We know from experience this simply doesn't work.
+
+Instead, our product, LunaSec, mitigates a vulnerability before anyone even knows of its existence.
+
+We want the team in charge of responding to security incidents to walk into any meeting or write any email and confidently
+say that "because our data is protected by LunaSec, a full compromise is not likely, and we have time to address the root
+cause".
+
+We expect you to be skeptical of any claim we may make, and for that reason that is why our code is fully open source.
```
Comment (freeqaz) [inline docs/blog/2021-12-28-log4j-hype-train.mdx:170]: Open Source * is the proper spelling to convey that it's permissively licensed.

File: docs/blog/2021-12-28-log4j-hype-train.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+Responding to the situation without considering how your own messaging can affect other teams can be more damaging than helpful.
+
+Having worked on security teams for large companies, we understand how important clear concise guidance is when executing
+a plan for mitigation. Changing what version of Log4j to update to mid-execution is a very costly process as it could leave
+some services, which were updated initially to address Log4Shell, now out of date with the latest recommendation.
+
+## Shameless Plug
+
+I have written about how security companies have been jumping at opportunities to make money in this time of panic, and the
+truth is so are we. There is, however, an important distinction to be made. LunaSec isn't a tool which just surfaces
```
Comment (freeqaz) [inline docs/blog/2021-12-28-log4j-hype-train.mdx:160]: Could be phrased a little better but I appreciate the honesty here haha

File: docs/blog/2021-12-28-log4j-hype-train.mdx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+## Fixing Vulnerabilities without the Panic
+
+Unclear messaging, incomplete fixes, and misleading severity have been prevalent through this entire patching process.
+Responding to the situation without considering how your own messaging can affect other teams can be more damaging than helpful.
+
+Having worked on security teams for large companies, we understand how important clear concise guidance is when executing
+a plan for mitigation. Changing what version of Log4j to update to mid-execution is a very costly process as it could leave
+some services, which were updated initially to address Log4Shell, now out of date with the latest recommendation.
+
+## Shameless Plug
```
Comment (freeqaz) [inline docs/blog/2021-12-28-log4j-hype-train.mdx:157]: This whole section could use a little cleanup but I like the sentiment of it.

**Review by freeqaz — CHANGES_REQUESTED** [review]
I'd like to tweak the copy a bit on this tomorrow. It's damn close though -- I think the structure and narrative is spot on. I just want to tighten up some of the language and help this post really shine.

**Comment by factoidforrest** [comment]
@breadchris I went to town on the intro a bit, please revert it or merge into your intro if you're not a fan.  The issues I was trying to fix are:

Incomplete/confusing context around this topic.  The hook made it unclear whether we were talking about issues in 2.16 or 2.17, and had no mention of the fact that the latest 2.17 ivuln is basically BS.  So i just went ahead and said it up top.  Did my best!

Made title more descriptive while retaining meaning

Recommend people go to 2.17 instead of 2.16, it's easier than explaining the caveats around what OS people are on.  The 2.16 vuln is soooomewhat legit so I just bumped it to 2.17.0 since thats what we are recommending elsewhere in our guides.

**Review by freeqaz — APPROVED** [review]

**Comment by factoidforrest** [comment]
Cool now just needs a few grammar/flow fixes before publish, I'll try to do it tonight or tomorrow

**Pull request merged**